tests/test_bot.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import gymnasium as gym
+import numpy as np
+
+from minigrid.utils.bot import Bot
+
+
+def test_bot():
+    # get all babyai envs
+    babyai_envs = []
+    for k_i in gym.envs.registry.keys():
+        if k_i.split("-")[0] == "BabyAI":
+            babyai_envs.append(k_i)
+
+    # random select one env
+    env = gym.make(np.random.choice(babyai_envs), render_mode="human")
```

Comment:
Could we `pytest.mark.parameterize` over all BabyAI rather than choose one at random